src/lib/auth.ts

```diff
@@ -0,0 +1,191 @@
+// export interface AuthContextValue {
+//   oAuthResult?: OAuthResult;
+//   requestInit?: RequestInit;
+// }
+
+// export function createAuthFromOAuthResult(oAuthResult: OAuthResult): AuthContextValue {
```

Comment:
why commented out?

Reply:
I forgot to remove this code. We don't need to use context since we pass the data through components props.

Even more: it's not even possible because the idea of a context is to call it directly from the components. But they are not aware of the context since they come from an external library. More useful when all the code is in the same place.